packages/flutter/test/material/material_test.dart

```diff
@@ -284,6 +285,30 @@ void main() {
     expect(pressed, isTrue);
   });
 
+  testWidgets('Material uses the dark SystemUIOverlayStyle when the background is light', (
+    WidgetTester tester,
+  ) async {
+    final ThemeData lightTheme = ThemeData(useMaterial3: true);
```

Comment:
super nit: `useMaterial3` defaults to true:) 
```suggestion
    final ThemeData lightTheme = ThemeData();
```

Reply:
done